custom-scope-handler/README.md

```diff
@@ -0,0 +1,29 @@
+## Custom Scope Handler
+
+### Introduction / Use case.
+This custom scope handler can be used to remove the scopes other than the allowed scopes from the token request. (This sample considers Scope_A, Scope_B and Scope_C as allowed scopes)
+
+### Applicable product versions.
+WSO2 IS 5.7.0
+
+### How to use.
+1. Build the custom-scope-handler using the command `mvn clean install`.
+2. Copy the CustomScopeHandler-1.0.0.jar located inside the target to the directory, <IS_Home>/repository/components/dropins
```

Comment:
Suggestion:
Copy the CustomScopeHandler-1.0.0.jar located inside the target to the directory, **<IS_Home>/repository/components/dropins**

Reply:
There is no difference between the suggestion and the existing line. Please verify the suggestion.